### DISCUSSION TITLE
How to enable rule expressions?

### DISCUSSION THREAD

**Discussion by vpmv** [firefly-iii/firefly-iii#8916] [Firefly III]
I've been breaking my mind on [rule expressions](https://docs.firefly-iii.org/how-to/firefly-iii/features/expressions/), only to find out they're purposely disabled (https://github.com/firefly-iii/firefly-iii/releases/tag/v6.1.11)
Is there a way to enable this feature? I really need it since I have a lot of non-descriptive payments that would benefit from easy conversion. 
On that matter it'd be nice to know how to combine literal strings with variables, e.g.: ="Groceries "~substr(description, 10)?

I'm thinking it might also be a good middle-ground solution to Payment Service Providers (PSPs) where multiple webshops are categorized into the same bank account. It would however, for that particular problem, be very nice to be able to remove all account details via a rule, and set the destination_account_name to a substr, but that's a separate issue.

**Comment by JC5** [comment]
Sorry, they are not enabled yet. The documentation page exists but it's not linked to anything. I guess the search picks it up?

Patience, I'll enabled it one of these days.

**Reply by floriskruisselbrink** [reply]
Ouch... maybe the documentation page should at least contain a big warning that this is not available (yet). 
I have been ripping my hear out for why it didn't work, even if I _exactly_ used the examples from the documentation.
And then I found this discussion 🙈

**Reply by JC5** [reply]
Sorry about that! :( i pushed those pages together with an unrelated grammar fix by mistake. I'll get right on it!
